src/elements/navigation/navigation-section/navigation-section.component.ts

```diff
@@ -218,10 +223,7 @@ class SbbNavigationSectionElement extends SbbUpdateSchedulerMixin(LitElement) {
 
   // Check if the trigger is valid and attach click event listeners.
   private _configureTrigger(): void {
-    const triggerElement = (
-      this.trigger instanceof HTMLElement ? this.trigger : this._triggerIdReferenceController.find()
-    ) as SbbNavigationButtonElement | null;
-
+    const triggerElement = this.trigger as SbbNavigationButtonElement | null;
```

Comment:
extra const could probably be removed, but casting then would be needed multiple times